docs/virtual_machines/device_status_on_Arm64.md

```diff
@@ -0,0 +1,25 @@
+# Device Status on Arm64
+
+This page is based on https://github.com/kubevirt/kubevirt/issues/8916
+
+Devices | Description | Status on Arm64
+-- | -- | --
+DisableHotplug |   | supported
+Disks | sata/ virtio bus | support virtio bus
+Watchdog | i6300esb | not support
```

Comment:
Can you review for consistency between "support" and "supported"? -ed forms are preferred.